src/common/njord.ts

```diff
@@ -192,111 +192,115 @@ const parseBalanceUpdate = ({
   };
 };
 
-export const transferCores = createAuthProtectedFn(
-  async ({
-    transaction,
-    entityManager,
-  }: TransferProps): Promise<TransferResult> => {
-    // TODO feat/transactions check if session is valid for real on whoami endpoint
+type TransferPropsRaw = Pick<TransferProps, 'transaction' | 'entityManager'>;
 
-    const njordClient = getNjordClient();
+export const transferCoresRaw = async ({
```

Comment:
Nothing changed here really, just the indents and the type of the props to receive (from TransferProps to TransferPropsRaw).

Reply:
Context guard is important, you can resolve user from worker side, it only requires `userId`

Reply:
It was asking for `isTeamMember`, and I think it loses its purpose if I just override that value.

Reply:
let me take a look, probably just types

Reply:
Amazing. Let me then revert this.

Reply:
Done! 👌